chia/_tests/blockchain/test_blockchain.py

```diff
@@ -308,7 +309,7 @@ async def test_unfinished_blocks(
         )
         npc_result = None
         if unf.transactions_generator is not None:
-            block_generator = await blockchain.get_block_generator(unf)
+            block_generator = await get_block_generator(blockchain.lookup_block_generators, unf)
```

Comment:
if we dont get here in the tests like the coverage says i think we should remove or fix, same for the two other cases in this file

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/18547